libraries/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -503,3 +508,29 @@ export class RushConfigurationProject {
     return isMain;
   }
 }
+
+export function validateRelativePathField(relativePath: string, name: string): void {
+  if (path.posix.isAbsolute(relativePath) || path.win32.isAbsolute(relativePath)) {
```

Comment:
Why not just `path.isAbsolute`?

Reply:
`path.isAbsolute` calls `path.posix.isAbsolute` on POSIX and `path.win32.isAbsolute` on Win32. `path.posix.isAbsolute('A:/b')` returns `false. `path.win32.isAbsolute('A:/b')` returns `true`.

Reply:
I'll add comments in the code for why.

Reply:
I guess because that would be system dependent?